Makefile

```diff
@@ -123,15 +123,15 @@ endif
 
 # Run unit tests.
 test:
-	go test -v ./auth/
+	go test -count=1 -v ./auth/
 
 # Run Hub REST API tests.
 test-api:
-	HUB_BASE_URL=${HUB_BASE_URL} go test -v ./test/api/...
+	HUB_BASE_URL=${HUB_BASE_URL} go test -count=1 -v ./test/api/...
```

Comment:
```-count=1``` disabled go test caching that is On by default and if code was not changes between test executions, tests are not executed, but result provided from cache (like ```PASS
ok  	github.com/konveyor/tackle2-hub/test/api/identity	(cached)
```), this is not wanted e.g. in case tests execution can be affected with environment variables like ```HUB_BASE_URL```.